src/styles/features/onboarding/onboarding.css

```
@@ -0,0 +1,510 @@
+@utility onboarding-select-wrapper {
+	@apply relative;
+}
+
+@utility onboarding-language-select {
+	@apply w-full px-3 py-2.5 pr-10 rounded-md text-sm border-none cursor-pointer appearance-none;
+	background-color: var(--color-bg-content);
+	color: var(--color-text);
+
+	&:focus {
+		@apply outline-none ring-2;
+		ring-color: var(--color-primary);
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)